app/lib/common/widgets/html_editor.dart

```diff
@@ -52,7 +53,8 @@ extension ActerEditorStateHelpers on EditorState {
   String intoHtml({
     AppFlowyEditorHTMLCodec? codec,
   }) {
-    return (codec ?? defaultHtmlCodec).encode(document);
+    final docHTML = (codec ?? defaultHtmlCodec).encode(document);
+    return HtmlUnescape().convert(docHTML);
```

Comment:
So there's a case where encoding this directly will result in HTML having escaped attributes. How it brings that kind of HTML? : if a user directly injects HTML code in editor instead of using editor features, so upon save it renders literal HTML strings. Having it converted to plain HTML prevents that insecurity. 

I didn't knew the rust method ```.contentMarkdown()``` was indeed specific for  handling only markdown content. As in previous version, this method was also handling HTML content...